OBSDOCS-3318 [DOC] Documentation for Perses Dashboard (#111992)
openshift/openshift-docs: modules/coo-perses-dashboards-create-ui.adoc

// Module included in the following assemblies:

// * observability/cluster_observability_operator/ui_plugins/perses-dashboard.adoc

:_mod-docs-content-type: PROCEDURE
[id="coo-perses-dashboards-create-ui_{context}"]
= Create a dashboard by using the web console

[role="_abstract"]

> You can create Perses dashboards by using the graphical editor in the {product-title} web console.

Reviewer: Not sure how is managed but here it says the cluster operator web console, but it should be the OpenShift web console.
```suggestion
You can create Perses dashboards by using the graphical editor in the OpenShift web console.
```

Author: Changing it to {ocp-product-title} as that's the correct attribute in COO docs.

.Prerequisites

* You have enabled the monitoring plugin.
* You have at least one datasource that exists in your target namespace, or a global datasource is available.

> * You have edit permissions in the target namespace.

Reviewer: ```suggestion
* You have Perses Dashboards edit permissions in the target namespace.
```

.Procedure

. In the {product-title} web console, navigate to *Observe* -> *Dashboards (Perses)*.

> . Select your target namespace from the project selector.

Reviewer: This is not required as the project is selected on the dialog that opens when clicking on create
```suggestion
```

> . Click *Create Dashboard*.

Reviewer: ```suggestion
. Click *Create*.
```

<img width="1199" height="801" alt="Image" src="https://github.com/user-attachments/assets/03b8987d-aebe-4f83-8b0e-86dfa515c8c9" />

. In the dashboard metadata fields, provide the following information:
+
* *Name*: A unique name for the dashboard.
* *Display Name*: An optional, user-friendly display name.
* *Description*: A description of the dashboard purpose.

> * *Tags*: Optional tags for organization and filtering.

Reviewer: ```suggestion
* *Project*: Select the target namespace from the dropdown.
```

. Add one or more panels to the dashboard:
+
.. Click *Add Panel*.

> .. Select a panel type: *Time series*, *Stat*, *Gauge*, *Bar chart*, or *Table*.

Reviewer: ```suggestion
.. Select from various panel types, including *Time series*, *Stat*, *Gauge*, *Bar chart*, *Table*, among others.
```

.. Configure the panel:
+
* *Title*: A descriptive title for the panel.
* *Datasource*: Select a datasource from the available list.
* *Query*: Enter a query in the appropriate query language (PromQL, LogQL, or TraceQL).
* *Visualization options*: Adjust display settings as required.

. Arrange panels by using drag-and-drop and resize handles.

. Optional: Configure dashboard variables:
+
.. Click *Dashboard settings* -> *Variables*.
.. Click *Add variable*.

> .. Enter the variable name, type, and query.

Reviewer: ```suggestion
.. Enter the variable name, type, and its type-specific query or value.
```

. Save the dashboard:
+

> .. Click *Save dashboard*.

Reviewer: ```suggestion
.. Click *Save*.
```

<img width="1199" height="789" alt="Image" src="https://github.com/user-attachments/assets/0a3ec534-f51f-4fee-9882-6112f9c3e109" />

.. Optional: Review the generated YAML.

> .. Click *Confirm*.

Reviewer: ```suggestion
```

.Verification

* The dashboard appears in the Perses dashboards list.
* Open the dashboard and verify that all panels display data correctly.
* If you configured variables, confirm that they function as expected.

[NOTE]
====
Dashboards created through the web console are automatically saved as `PersesDashboard` custom resources in your selected namespace.

> ====

Reviewer: This is currently not the case, the Perses dashboards created from the UI are not persisted as CRs. They are only persisted inside the Perses instance.
